packages/build/.eslintrc.js

```diff
@@ -63,6 +63,22 @@ module.exports = {
     },
   ],
   rules: {
+    'import/order': [
+      'error',
+      {
+        groups: [
+          'builtin',
+          'external',
+          'internal',
+          'parent',
+          'sibling',
+          'index',
+          'object',
+          'type',
+        ],
+        'newlines-between': 'always-and-inside-groups',
```

Comment:
Now that I look at it, I wish we went with `always`. 😶

Does `always-and-inside-groups` have any advantages over `always`? It seems to me that you'd always want to keep the groups together.

Reply:
I just ran autofix on master with `always` and it looks like there's a lot of files where the spaces seem to be arbitrarily injected from when the autofix was first ran.